src/main/java/controller/GetMessage.java

```diff
@@ -27,7 +27,6 @@
  * (1)  MessageId
  * (2)  ChatId
  * (3)  SentByCurrentUser
- * (4)  ContentType
```

Comment:
The serial number is now messed up.
Consider using the HTML tags `<ol>` and `<li>` instead.
They are allowed and supported in Javadoc.

Reply:
Done